build/localnet/README.md

```diff
@@ -206,6 +206,40 @@ The k8s manifests that `tilt` submits to the cluster can be found in [this direc
 
 Tilt continuously monitors files on local filesystem in [specific directories](Tiltfile#L27), and it rebuilds the binary and distributes it to the pods on every code change. This allows developers to iterate on the code and see the changes immediately (i.e. hot-reloading).
 
+## Debug with dlv
+
+### k8s LocalNet: Connect to a node debugging server
+
+LocalNet Pocket nodes have a dlv debugging server opened on port 7081. In order to connect a debugging client (VSCode, GoLand) and be able to set breakpoints, we need to setup a tunnel with the pod we want to debug.
+
+```
+kubectl port-forward validator-001-pocket-0 7081:7081
```

Comment:
Could you create a make target for this as well so people can run `make kube_debug`?